src/index.html

```diff
@@ -1,13 +1,77 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <div data-qa="card" class="card">
+
+      <div class="card__image-container">
```

Comment:
this container is redundant. Apply styles directly to an image
```suggestion
```

Reply:
I need container to center img
because I can't use hor... margins

Reply:
there shouldn't be any horizontal margins
add top padding to the card instead of top margin to the image
same here. add bottom padding to the card
Be consistent with your margins. Add only top or bottom, don't add both.

All is done...

Reply:
Use an image as block with margin
Example:
```
.image {
  display: block;
  height: ....;
  width: ....;
  margin: 0 auto 40px;
}
```